{"name": "test_ceshi_002", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 < 1", "trace": "self = <Test.TestCase.test_ceshi.TestCeshi object at 0x7f9bff12c2e0>\n\n    @allure.story(\"这是测试1<1失败的情况\")\n    def test_ceshi_002(self):\n        print(\"这是测试用例1\")\n>       assert 1 < 1\nE       assert 1 < 1\n\nTest/TestCase/test_ceshi.py:19: AssertionError"}, "start": 1650031069857, "stop": 1650031069857, "uuid": "e276559a-d415-4960-a3e5-da26a029aaf7", "historyId": "ab0990ae58dd4c039f227e0eb85aedc7", "testCaseId": "72f1c64968fc2a4c71c17db7a4a099d5", "fullName": "Test.TestCase.test_ceshi.TestCeshi#test_ceshi_002", "labels": [{"name": "story", "value": "这是测试1<1失败的情况"}, {"name": "feature", "value": "ceshi"}, {"name": "parentSuite", "value": "Test.TestCase"}, {"name": "suite", "value": "test_ceshi"}, {"name": "subSuite", "value": "TestCeshi"}, {"name": "host", "value": "fv-az102-366"}, {"name": "thread", "value": "2367-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test.TestCase.test_ceshi"}]}